consensus/istanbul/backend/engine.go

```diff
@@ -357,7 +357,7 @@ func (sb *backend) verifyCommittedSeals(chain consensus.ChainReader, header *typ
 	}
 
 	// The length of validSeal should be larger than number of faulty node + 1
-	if validSeal <= 2*snap.ValSet.F() {
+	if validSeal < istanbulCore.RequiredMessageCount(snap.ValSet, chain.Config().IsCancunForkEnabled(header.Number)) {
```

Comment:
I think `2F()` can't be directly changed to `RequiredMessageCount` since it differs for small validators' cases. Is it okay?

Reply:
Oh, it was updated. But I think we still need to consider `n=6` case.

Reply:
Let's try a non hard-fork-required one